File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Payments/Payment.cs
using Ohtap.ECommerce.Domain.Models.Objects.Products;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Currencies;

namespace Ohtap.ECommerce.Domain.Models.Objects.Payments
{
    public class Payment : PaymentBasic, IPayment
    {
        public virtual Banks.IBankData Bank { get; set; }
        public virtual Customers.ICustomerData Customer { get; set; }
        public virtual ICurrencyData Currencya { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Payments/Payment .php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Payments;

class Payment  implements IPayment
{

    public $Customers;//Customer
    public $Bank;//Bank
    public $Currency;//Currency
    public $Types;//PaymentTypes
    public $Amount;
    public $Title;
    public $Name;
    public $Value;
    public $Description;
    public $State;
    public $OrderNo;
    public $ID;
}